frontend/packages/integration-tests-cypress/tests/crud/secrets/image-pull.cy.ts

```diff
@@ -130,10 +134,21 @@ describe('Image pull secrets', () => {
     secrets.enterSecretName(uploadConfigFileImageSecretName);
     cy.byTestID('console-select-auth-type-menu-toggle').click();
     cy.byTestDropDownMenu('config-file').click();
-    cy.byLegacyTestID('file-input-textarea').type(JSON.stringify(configFile), {
+
+    // Type the JSON slowly to give validation time to process
+    const configJson = JSON.stringify(configFile);
+    cy.byLegacyTestID('file-input-textarea').clear().type(configJson, {
       parseSpecialCharSequences: false,
+      delay: 50, // Slower typing to give validation time
```

Comment:
should we be debouncing the input instead of typing slower? is there lag with the delay?

Reply:
Could pasting the input be faster? Something like:
```
const configJson = JSON.stringify(configFile);
    cy.byLegacyTestID('file-input-textarea')
      .invoke('val', configJson)
      .trigger('input', { bubbles: true })
      .trigger('change', { bubbles: true });
```